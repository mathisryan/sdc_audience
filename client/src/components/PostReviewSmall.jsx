import React from 'react';

class PostReviewSmall extends React.Component {
  constructor(props) {
    super(props);
  }
  render() {
    return (
    <div className='adduser-container-sm'>  
      <div className="add-wrapper-sm">
        <i className="fas fa-plus"></i>
        <br/>
        <div className="wantToSee-sm mobile-btn" value="+">Want to See</div>
      </div>
      <div className="add-wrapper-sm">
        <i className="fas fa-minus"></i>
        <br/>
        <div className="interested-sm mobile-btn" value="-">Not Interested</div>
      </div>
      <div className="add-wrapper-sm">
        <i className="fas fa-star"></i>
        <br/>
        <div className="addrating-sm mobile-btn" value="1">Add Rating</div>
      </div>
    </div>

    )
  }
}

export default PostReviewSmall;