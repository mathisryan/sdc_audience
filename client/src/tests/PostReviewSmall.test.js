import React from 'react';
import { shallow, mount } from 'enzyme';
import PostReviewSmall from '../components/PostReviewSmall';
import './setupTest'

describe('PostReviewSmall', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<PostReviewSmall debug />);
    expect(component).toMatchSnapshot();
  });

  it('should have correct rating text', () => {
    const wrapper = mount(<PostReviewSmall/>);
    expect(wrapper.find('div').at(2).text()).toEqual('Want to See');
  });
});
