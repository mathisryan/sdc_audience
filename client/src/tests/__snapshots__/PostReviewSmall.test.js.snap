// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostReviewSmall should render correctly in "debug" mode 1`] = `
<div
  className="adduser-container-sm"
>
  <div
    className="add-wrapper-sm"
  >
    <i
      className="fas fa-plus"
    />
    <br />
    <div
      className="wantToSee-sm mobile-btn"
      value="+"
    >
      Want to See
    </div>
  </div>
  <div
    className="add-wrapper-sm"
  >
    <i
      className="fas fa-minus"
    />
    <br />
    <div
      className="interested-sm mobile-btn"
      value="-"
    >
      Not Interested
    </div>
  </div>
  <div
    className="add-wrapper-sm"
  >
    <i
      className="fas fa-star"
    />
    <br />
    <div
      className="addrating-sm mobile-btn"
      value="1"
    >
      Add Rating
    </div>
  </div>
</div>
`;
